sandbox_osteinnes/MarioBros_libGDX/core/src/com/mygdx/testgamev1/MarioBros.java

```diff
@@ -52,22 +64,70 @@ public SpriteBatch getBatch() {
 	 * @return Virtual Width
 	 */
 	public int getvWidth() {
-		return this.V_WIDTH;
+		return V_WIDTH;
 	}
 
 	/**
 	 * Returns the virtual height of the game.
 	 * @return Virtual Height
 	 */
 	public int getvHeight() {
-		return this.V_HEIGHT;
+		return V_HEIGHT;
 	}
 
 	/**
 	 * Returns the pixels per meter value of the game
 	 * @return Pixels per meter
 	 */
 	public float getPixelsPerMeter() {
-		return this.pixelsPerMeter;
+		return pixelsPerMeter;
+	}
+
+	/**
+	 * Returns the default bit constant.
+	 * @return the default bit constant.
+	 */
+	public static short getDefaultBit() {
+		return DEFAULT_BIT;
```

Comment:
Could just make them public static final int, without getX() methods

Reply:
Yes, I have made this habit where I never make public fields in a class. I make getter methods instead. But when a field is final, that should not be necessary I suppose.